src/queue.rs

```diff
@@ -250,6 +252,11 @@ where
         self.signal_after_end = signal_after_end;
         Ok(())
     }
+
+    /// 200 frames of silence
+    fn silent_span_length(&self) -> usize {
+        200 * self.channels() as usize
```

Comment:
self.channels() will return 1 when we're either playing silence or at the start of the queue with the `Empty` source.